nodejs/src/wrapper.ts

```diff
@@ -66,7 +71,8 @@ async function initializeProvider() {
     detectors: [awsLambdaDetector, envDetector, processDetector],
   });
   const tracerConfig: NodeTracerConfig = {
-    resource
+    resource,
+    forceFlushTimeoutMillis: forceFlushTimeoutMillis
```

Comment:
nit, can omit the value:

```suggestion
    forceFlushTimeoutMillis,
```